__tests__/app.test.js

```diff
@@ -152,3 +153,81 @@ describe("GET /api/users", () => {
     });
   });
 });
+
+/* ***************GET ARTICLE BY ID WITH COMMENT COUNT****************/
+
+describe("GET /api/articles/:article_id returns correct article", () => {
+  test("endpoint returns correct article", async () => {
+    const results = await request(app).get("/api/articles/1").expect(200);
+    expect(results.body.article.comment_count).toBe(11);
+  });
+});
+
+/* ***************GET ARTICLE DESC DATE****************/
+
+describe("GET /api/articles returns all articles in descending order", () => {
+  test("endpoint returns array of articles in descending order", async () => {
+    const results = await request(app).get("/api/articles").expect(200);
+    expect(results.body.article).toBeSortedBy("created_at", {
+      descending: true,
+    });
+  });
+  test("endpoint returns array of articles with comment count", async () => {
```

Comment:
This seems like it's in the wrong section? Shouldn't it be higher up with the other comment count test?